README.md

```diff
@@ -22,6 +22,18 @@ of the following ways:
 - Create a [Github issue](https://github.com/pulibrary/dls-handbook/issues)
 - Message us in #digital_library in PULibrary's Slack
 
+## Our Vision
+
+DLS has worked with its partners to define a vision for the Digital Library in
+the context of Princeton University Library's [Mission, Vision, and North Star
+Statements](https://library.princeton.edu/about) to help us situate
```

Comment:
I like this! Since we have "to define... to help" what if we made it "that will help" instead.